langchain4j-azure-open-ai/src/test/java/dev/langchain4j/model/azure/InternalAzureOpenAiHelperTest.java

```diff
@@ -32,11 +33,26 @@ void setupOpenAIClientShouldReturnClientWithCorrectConfiguration() {
         Integer maxRetries = 5;
         boolean logRequestsAndResponses = true;
 
-        OpenAIClient client = InternalAzureOpenAiHelper.setupOpenAIClient(endpoint, serviceVersion, apiKey, timeout, maxRetries, null, logRequestsAndResponses);
+        OpenAIClient client = InternalAzureOpenAiHelper.setupSyncClient(endpoint, serviceVersion, apiKey, timeout, maxRetries, null, logRequestsAndResponses);
 
         assertThat(client).isNotNull();
     }
 
+    @Test
+    void setupOpenAIAsyncClientShouldReturnClientWithCorrectConfiguration() {
+        String endpoint = "test-endpoint";
+        String serviceVersion = "test-service-version";
+        String apiKey = "test-api-key";
+        Duration timeout = Duration.ofSeconds(30);
+        Integer maxRetries = 5;
+        boolean logRequestsAndResponses = true;
+
+        OpenAIAsyncClient client = InternalAzureOpenAiHelper.setupAsyncClient(endpoint, serviceVersion, apiKey, timeout, maxRetries, null, logRequestsAndResponses);
+
+        assertThat(client).isNotNull();
+    }
+
+
```

Comment:
Please extend `AzureOpenAiStreamingChatModelIT` to make sure both sync and async clients work fine